src/components/Button/button.css

```diff
@@ -28,7 +28,14 @@
   @apply lil-w-16 lil-h-16 lil-rounded-full lil-bg-grey-150 lil-cursor-pointer lil-items-center lil-justify-center lil-flex lil-text-s-24;
 }
 
+.lil-btn-round-white {
```

Comment:
🤔  this is for ?

Reply:
new pages have different design for button